srcpkgs/kvmtool/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'kvmtool'
+pkgname=kvmtool
+version=3.18.0
+revision=1
+wrksrc=HEAD
+build_style=gnu-makefile
+make_use_env=0
+hostmakedepends=""
+makedepends=""
+depends=""
+short_desc="Native Linux KVM tool"
+maintainer="auser509 <auser509@gmail.com>"
+license="GPL-2"
+homepage="https://git.kernel.org/pub/scm/linux/kernel/git/will/kvmtool.git/"
+distfiles="https://git.kernel.org/pub/scm/linux/kernel/git/will/kvmtool.git/snapshot/HEAD.tar.gz"
```

Comment:
We prefer tagged releases, or at least pin it to a specific git commit. This will break on the next commit to the project